doc/source/changes.rst

```diff
@@ -12,14 +12,29 @@ to adapt in your project configuration, database etc. when upgrading to a new ve
 Version 1.7.2 (CURRENT DEV)
 ---------------------------
 This is the current development version, not yet released.
-Changes thus far are relevant only for users of MapFish Print.
+
+Oereblex related changes
+^^^^^^^^^^^^^^^^^^^^^^^^
+Pyramid Oereb now supports and uses by default geoLink schema version 1.2.0:
+
+ * New doctype 'notice' (will be classified as 'HintRecord'). If you want to add related notice as
+   additional legal provision directly to the public law restriction, You can set the new oereblex
+   'related_notice_as_main' flag in the config of the project.
+ * 'Notice' can have no authority nor enactment_date. In this case, the enactment date will be
+   '01.01.1970' and the authority '-'.
+ * 'Notice' can have no authority_at_web. That was not supported by MapFish Print. If you use MapFish Print
+   with Oereblex 1.2.0, you must update your MapFish Print instance.
+ * New document attribute 'language' and file attribute 'description' are currently not used by
+   pyramid_oereb but are now available to custom code, for example for document title generation.
 
 MapFish Print related changes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 The inclusion of all geometry data in the print payload is now configurable (#1006).
 MapFish Print should set the print configuration parameter ```with_geometry``` to ```False```
 to improve performance.
 
+It's now possible to print report with missing OfficeAtWeb information in documents.
```

Comment:
Small change suggestion:

It is now possible to print reports with missing OfficeAtWeb information in documents (OfficeAtWeb is an optional attribute in the specification).